app/views/transactions.html

```diff
@@ -76,7 +77,7 @@ <h3 style="margin: 20px;" ng-if="transactionsSelected.length < defaultBulkRerunL
                 <tr class="table-list" ng-repeat="transaction in transactions" ng-click='viewTransactionDetails("transactions/" + transaction._id, $event)'>
                   <td class="transactionsCheckboxTD">
                     <label>
-                      <input style="float: left" type="checkbox" ng-change="toggleTransactionSelection(transaction._id)" ng-if="( channelsMap[transaction.channelID].rerun || rerunAllowedAdmin ) && transaction.canRerun" ng-checked="checkbox.checkAll" ng-model="checkBox" value="transaction._id"/>
+                      <input style="float: left" type="checkbox" ng-change="toggleTransactionSelection(transaction._id)" ng-if="( channelsMap[transaction.channelID].rerun === true ) && transaction.canRerun" ng-checked="checkbox.checkAll" ng-model="checkBox" value="transaction._id"/>
```

Comment:
You probably don't need the `=== true` here. You should be able to do something like `channelsMap[transaction.channelID].rerun && transaction.canRerun`